src/main/resources/messages.properties

```diff
@@ -1,9 +1,11 @@
 # Validation messages for Entity fields
+assignment.not.exists = Doctor is not yet assigned to patient
```

Comment:
Change this message "Doctor is not yet assigned to patient" to
The doctor has not been assigned to the patient.

Reply:
Resolved